test/test-execute/exec-systemcallfilter-override-error-action.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Test for SystemCallFilter with specific kill action overriding default errno action
+
+[Service]
+ExecStart=/usr/bin/python3 -c 'import os\ntry: os.uname()\nexcept Exception as e: exit(e.errno)'
+Type=oneshot
+SystemCallFilter=~uname:kill
```

Comment:
This test needs the other commit to function, so it should be with the later commit.

Reply:
squashed commits